core/modules/widgets/navigator.js

```diff
@@ -539,8 +540,8 @@ NavigatorWidget.prototype.handleImportTiddlersEvent = function(event) {
 	// Save the $:/Import tiddler
 	newFields.text = JSON.stringify(importData,null,$tw.config.preferences.jsonSpaces);
 	this.wiki.addTiddler(new $tw.Tiddler(importTiddler,newFields));
-	// Update the story and history details
-	var autoOpenOnImport = event.autoOpenOnImport ? event.autoOpenOnImport : this.getVariable("tv-auto-open-on-import");  
+	// Update the story and history details.
```

Comment:
most of the time full-stops are missing intentionally

See: https://github.com/Jermolene/TiddlyWiki5/issues/519
and: https://en.wikipedia.org/wiki/Full_stop#In_text_messages

Reply:
Thanks. That was a mistake. I fixed it.